week5.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>In question 4, between steps 2 and 3, you cancel piR^2. But what happens to the 4 in the denominator (with 4piR^2?) I'm probably missing something really obvious...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-jolsenvaldez/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>